hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/ingest/IngestCheckerTest.java

```diff
@@ -216,13 +224,25 @@ void testParseAndCheckWithInactivePlatformFails(final PlatformStatus status) {
             if (status != PlatformStatus.ACTIVE) {
                 // Given a platform that is not ACTIVE
                 when(currentPlatformStatus.get()).thenReturn(status);
+                given(blockStreamManager.hasLedgerId()).willReturn(true);
                 // When we try to parse and check a transaction, it should fail because the platform is not active
-                assertThatThrownBy(() -> subject.checkNodeState())
+                assertThatThrownBy(() -> subject.verifyPlatformActive())
                         .isInstanceOf(PreCheckException.class)
                         .has(responseCode(PLATFORM_NOT_ACTIVE));
                 verify(opWorkflowMetrics, never()).incrementThrottled(any());
             }
         }
+
+        @Test
+        @DisplayName("Even if the platform is not ACTIVE, waits for ledger id to be available")
```

Comment:
Should this be "Even if the platform is ~~not~~ ACTIVE, waits for ledger id..." ?